<?xml version="1.0"?>

<launch>

  <!--  URG Settings -->
  <arg name="~ip_address" default="192.168.10.53" />
  <arg name="~frame_id" default="wheels_base_laser_link" />

  <node name="urg_node" pkg="urg_node" type="urg_node"
        respawn="true">
    <param name="~ip_address" value="$(arg ~ip_address)" />
    <param name="~frame_id" value="$(arg ~frame_id)" />

    <!-- +-115[deg] -->
    <param name="~angle_min" value="-2.007128639793479" />
    <param name="~angle_max" value="2.007128639793479" />

  </node>

  <!-- DualShock3 Settings -->
  <!-- <arg name="joy_config" default="ps3-holonomic" /> -->
  <!-- <arg name="joy_dev" default="/dev/input/js0" /> -->
  <!-- <arg name="config_filepath" default="$(find teleop_twist_joy)/config/$(arg joy_config).config.yaml" /> -->

  <!-- <node pkg="joy" type="joy_node" name="joy_node"> -->
  <!--   <param name="dev" value="$(arg joy_dev)" /> -->
  <!--   <param name="deadzone" value="0.3" /> -->
  <!--   <param name="autorepeat_rate" value="20" /> -->
  <!-- </node> -->

  <!-- <node pkg="teleop_twist_joy" name="teleop_twist_joy" type="teleop_node"> -->
  <!--   <rosparam command="load" file="$(arg config_filepath)" /> -->
    <!-- <param name="~enable_button" value="10" />  --> <!-- L1 Button -->
    <!-- <param name="~enable_turbo_button" value="8" />  --> <!-- L2 Button -->
  <!--   <param name="~scale_linear" value="0.3" /> -->
  <!--   <param name="~scale_angular" value="0.5" /> -->
  <!--   <param name="~scale_linear_turbo" value="0.5" /> -->
  <!-- </node> -->

</launch>
